decidim-meetings/app/models/decidim/meetings/meeting.rb

```diff
@@ -95,6 +95,7 @@ class Meeting < Meetings::ApplicationRecord
 
       TYPE_OF_MEETING.each do |type|
         scope type.to_sym, -> { where(type_of_meeting: type.to_sym) }
+        scope "not_#{type}".to_sym, -> { where.not(type_of_meeting: type.to_sym) }
```

Comment:
💬 I think this is ok and I'd not change it now, but I feel we are rewriting enums here. 😕